data/remnant outfits.txt

```diff
@@ -1,4 +1,5 @@
 # Copyright (c) 2017 by Michael Zahniser
+# Copyright (c) 2019 by Zitchas
```

Comment:
Pretty sure copyright declarations require real names.

Reply:
Are you sure that's not my real name? (or at least a portion thereof?)

Either way, last year I was told by someone that we could basically put whatever we wanted in terms of copyright attribution. At the end of the day, this is basically along the same lines as authors using noms-de-plumes, and that's a practice that is quite firmly established in legal precedence. Admittedly, I have no expertise in the subject, but it is a long standing practice.

Reply:
And I've mentioned it before, but I don't see one small addition as warranting listing yourself as having the copyright. You didn't do this for the ships file, or the normal remnant file. Why now?

Reply:
Probably should have for the Ships one, but I didn't for the Remnants.txt because I haven't made significant changes to it, and don't foresee doing so going forward. I'm definitely planning on quite a bit of new stuff for both the Ships and Outfits. But I'm fine with taking that out and putting it on in the future when there's more content.